src/theme/index.ts

```diff
@@ -23,14 +25,18 @@ const createPalette = (options: PaletteOptions) => {
     },
     secondary: {
       main: '#ff5a5f',
+      dark: midGrey,
```

Comment:
This doesn't make sense. light & dark should be different _shades_ of the same _hue_.

Reply:
yeah this is an oxymoron! Thanks - will update to what it was before.